Sources/APPX.cpp

```diff
@@ -102,6 +104,15 @@ namespace appx {
             for (const auto &fileNamePair : fileNames) {
                 const std::string &archiveName = fileNamePair.first;
                 const std::string &fileName = fileNamePair.second;
+
+                const std::string suffix = "AppxBundleManifest.xml";
+                if (bundle &&
+                        suffix.size() < archiveName.size() &&
+                        std::equal(suffix.rbegin(), suffix.rend(), archiveName.rbegin())) {
```

Comment:
We are doing the same thing with AppxManifest.xml for .APPX archives. Basically, these two are the only files with hardcoded name that we need to provide externally (i.e. that we do not generate by the appx executable) and we make sure they are provided in corresponding python scripts. What we can do maybe is exit the app and print out the message in case these files are not provided from the command line.